docker/scripts/dev_start.sh

```diff
@@ -268,17 +268,22 @@ function do_docker_image_inspect() {
 function do_docker_pull() {
     IMG=$1
     if [ "$NO_PULL_IMAGE" = "yes" ];then
-        echo "Skipping pull docker image for $IMG"
+        info "Skipping pull docker image for $IMG"
         # check for local existence if we skip
         do_docker_image_inspect $IMG
     else
-        info "Start pulling docker image $IMG ..."
-        docker pull $IMG
-        if [ $? -ne 0 ];then
-            error "Failed to pull docker image : $IMG"
-            exit 1
+        docker images --format "{{.Repository}}:{{.Tag}}" | grep "^${IMG}$" > /dev/null
+        if [ $? -eq 0 ]; then
```

Comment:
This PR is problematic: Suppose docker registry  has say, `docker.io/apolloauto/apollo:local_third_party_volume-x86_64-latest` updated. A user who has the `apolloauto/apollo:local_third_party_volume-x86_64-latest` image once pulled will **NEVER** get updated unless he/she removed his/her local copy manually.